zerver/lib/integrations.py

```diff
@@ -90,6 +90,11 @@ def __init__(self, name, display_name=None, logo=None, logo_alt=None, git_url=No
         self.git_url = git_url
         super(HubotLozenge, self).__init__(name, name, logo, display_name=display_name)
 
+class GithubIntegration(WebhookIntegration):
```

Comment:
this is reasonable, but deserves explanatory comments both here and in `urls.py` :)